app/src/test/java/org/fossasia/openevent/app/presenter/EventDetailPresenterTest.java

```diff
@@ -192,4 +190,23 @@ public void shouldNotAccessView() {
 
         verifyZeroInteractions(eventDetailView);
     }
+
+    @Test
+    public void shouldHideProgressbarCorrectly() {
+        when(eventRepository.getAttendees(id, false))
+            .thenReturn(Observable.fromIterable(attendees));
+
+        when(eventRepository.getEvent(id, false))
+            .thenReturn(Observable.just(event));
+
+        InOrder inOrder = Mockito.inOrder(eventRepository, eventDetailView);
+
+        eventDetailPresenter.loadAttendees(id, false);
+        eventDetailPresenter.loadTickets(id, false);
+
+        while(eventDetailPresenter.getProgress() < 2) {
```

Comment:
This is not the correct way to write the test. Load both the attendees and you should not know that the progress should be less than 2 and I have to wait it it becomes so. Besides, all the work is synchronous in threads, so it should work after removing this